.github/tests/dependencies/postgresql.yaml

```diff
@@ -6,3 +6,8 @@ primary:
       drop: [ALL]
     seccompProfile:
       type: RuntimeDefault
+auth:
+  database: spire-server
+  username: spire
+  password: sp1ff3Test
+  postgresPassword: sp1ff3TestPassword
```

Comment:
Did this because it would complicate the test setup. This is reused in both examples and ci/*-values.yaml tests.

Making this on variable generated password would require again whole bunch of plumbing on the chart-testing tool.

Reply:
It is a regression of the tests though. So we should make sure we retain the same functionality....

We could do that part of the tests using unit testing instead, to ensure that passwords are passed through to the generated config. Would that work? We can adapt https://github.com/spiffe/helm-charts/pull/260/files#diff-a86124ab60e89d8dbec3b476b5a21077f58272f61f2007b0136c84e609738bb2 to do that kind of test, then I'm ok with doing a hardcoded pw test in the integration test.

Reply:
What functionality are we not testing anymore? I only moved them? tests are still the same.

Reply:
The hardcoded value proofs the same as it wouldn't work without defining those values either. No need to make things more complex then necessary.